src/components/common/Dialog/BasicDialog.tsx

```
@@ -27,9 +27,14 @@
   return (
     <Dialog open={open} onOpenChange={onOpenChange}>
       {triggerButton && <DialogTrigger asChild>{triggerButton}</DialogTrigger>}
-      <DialogContent className='sm:max-w-[425px] md:max-w-[1000px] dark:bg-neutral-900 dark:text-white dark:border-neutral-700'>
+      <DialogContent
+        className='sm:max-w-[425px] md:max-w-[1000px] dark:bg-neutral-900 dark:text-white dark:border-neutral-700'
+        aria-labelledby='basic-dialog-title' // Link to the dialog's title for accessibility
+      >
         <DialogHeader>
-          <DialogTitle className='dark:text-white'>{title}</DialogTitle>
+          <DialogTitle id='basic-dialog-title' className='dark:text-white'> // Add ID for aria-labelledby
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)